front/components/assistant/conversation/input_bar/editor/useCustomEditor.tsx

```diff
@@ -211,7 +190,11 @@ const useCustomEditor = ({
     StarterKit.configure({
       hardBreak: false, // Disable the built-in Shift+Enter. We handle it ourselves in the keymap extension
       strike: false,
+      bold: false, // Disable default bold, we use a custom one
+      italic: false, // Disable default italic, we use a custom one
     }),
+    CustomBold,
+    CustomItalic,
```

Comment:
😢 we had custom markdown for bold and italic, so we need to be backward compatible

Reply:
it's serialized as `*` not `_`, and the extra space at the beginning was not handled ex: `_ here_`